iree/compiler/Conversion/LinalgToLLVM/Passes.cpp

```diff
@@ -77,13 +68,12 @@ void addLinalgToLLVMPasses(OpPassManager &passManager,
 
 void buildLLVMTransformPassPipeline(OpPassManager &passManager,
                                     LLVMCodegenOptions options) {
-  OpPassManager &nestedModulePM = passManager.nest<ModuleOp>();
-
-  nestedModulePM.addPass(createInlinerPass());
-
-  // HLO -> Linalg on buffers.
   if (options.usingLinalgOnTensors) {
-    nestedModulePM.addNestedPass<FuncOp>(createLinalgVectorizePass());
+    passManager.addPass(createMaterializeCPULaunchConfigurationPass());
+    OpPassManager &nestedModulePM = passManager.nest<ModuleOp>();
+    nestedModulePM.addPass(createInlinerPass());
```

Comment:
Probably better to move the inline pass before. Any reason why you moved it after?

Reply:
As we discussed before we want to get these constant materialized as early as possible and substituting the tiling constants right after the backend is picked.

Reply:
Yes, I remember. Just saying maybe do this after inlining. Actually probably dont need inlining on the Linalg on tensors path.

Reply:
Yeah, good point! looking at the IR dump of mobile bert, seems like the inliner is totally no-op here. will remove it

Reply:
I can send a followup PR to remove the `Ininer`